homeassistant/components/repairs/issue_handler.py

```diff
@@ -124,6 +126,7 @@ def create_issue(
     *,
     breaks_in_ha_version: str | None = None,
     is_fixable: bool,
+    is_persistent: bool,
```

Comment:
Why make this required? 

I think we should default it to false (and maybe we should consider doing the same thing for `is_fixable` by default).

```suggestion
    is_persistent: bool = False,
```

Not doing that, would be a breaking change (as this is now a "public" API with the release of 2022.8.0).